### PR TITLE
Add LeapMicro 5.4 pipelines

### PR DESCRIPTION
**Pull request by lkocman**
* disable any previous pipenlines, we only produce images for the currently developed ones

### CODE REVIEW

**Comment by lkocman** [comment]
OpenQA setup https://progress.opensuse.org/issues/124134

**Review by Vogtinator — CHANGES_REQUESTED** [review]
You deleted `totestmanager.gocd.yaml.erb`...

**Comment by lkocman** [comment]
Ah thanks for catching this. Accidental typo ">" at git add >$filename  is why it was erased based on history. Fixed

**Review by Vogtinator — CHANGES_REQUESTED** [review]

File: gocd/totestmanager.gocd.yaml.erb [hunk 1]
```diff
@@ -13,10 +13,8 @@ pipelines:
       openSUSE:Leap:15.5:Images
       openSUSE:Leap:15.5:ARM
       openSUSE:Leap:15.5:ARM:Images
-      openSUSE:Leap:Micro:5.2
-      openSUSE:Leap:Micro:5.2:Images
-      openSUSE:Leap:Micro:5.3
-      openSUSE:Leap:Micro:5.3:Images
+      openSUSE:Leap:Micro:5.4
+      openSUSE:Leap:Micro:5.4444:Images
```
[the remark this answers is not in the document]
Comment (lkocman) [reply gocd/totestmanager.gocd.yaml.erb]: Yeah it would take us only 2222 years (two releases a year) :-)

**Pull request merged**